suite/tests/linux/syscall_pwait.c

```diff
@@ -93,119 +126,141 @@ main(int argc, char *argv[])
 
     INIT();
 
-    sleeptime.tv_sec = 0;
-    sleeptime.tv_nsec = 500 * 1000 * 1000;
+    sleeptime.tv_sec = 1;
+    sleeptime.tv_nsec = 0;
 
     intercept_signal(SIGUSR1, (handler_3_t)signal_handler, true);
     intercept_signal(SIGUSR2, (handler_3_t)signal_handler, true);
     print("handlers for signals: %d, %d\n", SIGUSR1, SIGUSR2);
     sigemptyset(&new_set);
+    sigaddset(&new_set, SIGUSR2);
```

Comment:
A comment would help explain why it is important to block both.

Reply:
Done.